module/modmanager/manager.go

```diff
@@ -87,12 +94,19 @@ type addedResource struct {
 
 // Manager is the root structure for the module system.
 type Manager struct {
-	mu                      sync.RWMutex
-	logger                  logging.Logger
-	modules                 map[string]*module
-	parentAddr              string
-	rMap                    map[resource.Name]*module
-	untrustedEnv            bool
+	mu           sync.RWMutex
+	logger       logging.Logger
+	modules      map[string]*module
+	parentAddr   string
+	rMap         map[resource.Name]*module
+	untrustedEnv bool
+	// viamHomeDir is the absolute path to the viam home directory. Ex: /home/walle/.viam
+	// it is empty if the modmanageroptions.Options.viamHomeDir was empty
```

Comment:
IIUC, this should only be empty as a convenience for testing? If that's true, I think it's preferable to call that out:

```
- it is empty if the modmanageroptions.Options.viamHomeDir was empty
+ `viamHomeDir` may only be the empty string in testing
```